### DISCUSSION TITLE
Can we replace ServiceNow CMDB with Netbox

### DISCUSSION THREAD

**Discussion by pravyn** [netbox-community/netbox#13687] [General]
Hello,

i' am very new to Netbox so thought Netbox community can help me on this.

We have 8000+ virtual machines in ServiceNow CMDB currently. Due to ITOM ServiceNow discovery licensing issue we want to get away from that product and move to an open source solution. Will netbox fit to our need ??? like doing inventory of 8000+ VM;s....etc.

Thanks,
Pravyn

**Comment by jcollie** [comment]
The answer is maybe. It really depends on what information that you want to store. Out of the box NetBox only captures very basic data about virtual machines. NetBox is very customizable though. You'll also need to build your own tools for actually using the data stored in NetBox to configure virtual machines.
